drivers/hmis/app/models/hmis/reminders/reminder_generator.rb

```diff
@@ -72,7 +80,9 @@ def last_assessment_date(enrollment:, stages:, wip:)
         found = assessments.detect do |r|
           wip.include?(r.wip) && stages.include?(r.data_collection_stage)
         end
-        found&.assessment_date
+        return unless found
+
+        [found.assessment_date, found]
```

Comment:
can we just return found rather than a tuple? Also suggest rename this method since it's no longer returning the date?
```diff
-  last_assessed_on, = last_assessment_date(enrollment: enrollment, stages: [:annual_assessment], wip: [false])
+  last_assessed_on = last_assessment(enrollment: enrollment, stages: [:annual_assessment], wip: [false])&.assessment_date
```

Reply:
definitely, fixed